Cargo.toml

```diff
@@ -29,7 +29,8 @@ num_cpus        = "1.12.0"
 rand            = "0.8.1"
 reqwest         = { version = "0.11.0", default-features = false, features = ["blocking", "rustls-tls" ] }
 ring            = "0.16.12"
-rpki            = { version = "0.12.2", features = [ "repository", "rrdp", "rtr", "serde" ] }
+routecore       = "0.1.1"
+rpki            = { git = "https://github.com/NLnetLabs/rpki-rs.git", branch = "main", features = [ "repository", "rrdp", "rtr", "serde", "slurm" ] }
```

Comment:
Shouldn't this be  = "0.14.1"?

Reply:
I had to add a few things to rpki to make it work with Routinator, so we need the main branch until there is 0.14.2.